src/core/create_compilers/RollupCompiler.ts

```diff
@@ -154,11 +158,40 @@ export default class RollupCompiler {
 
 		const that = this;
 
-		// TODO this is hacky. refactor out into an external rollup plugin
-		(mod.plugins || (mod.plugins = [])).push(css_chunks({ injectImports: true }));
-		if (!/[\\/]client\./.test(entry_point)) {
-			return mod;
-		}
+		/**
+		 * Track css code hashes for proper chunk hashing
+		 */
+		const css_hashing: Plugin = {
+			name: 'sapper-css-hashing',
+			augmentChunkHash(this: PluginContext, chunk: PreRenderedChunk) {
+				let hash: Hash;
+
+				const module_ids = Object.keys(chunk.modules).map(module_id => ({ module_id, module_path: path.parse(module_id) }));
+				module_ids.forEach(({ module_id, module_path }) => {
+					if (that.css_hashes[module_id]) {
+						const file = `${module_path.dir}/${module_path.name}`;
+						// ignore svelte css files created with `emitCss` enabled
```

Comment:
this comment could be expanded. why ignore those files and not other css files? also, how does this check work?